app/Database/Migrations/2024-08-10-143100_AddUser.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Database\Migrations;
+
+use CodeIgniter\Database\Migration;
+use CodeIgniter\Database\RawSql;
+
+class AddUser extends Migration
+{
+
+    /**
+     * @inheritDoc
+     */
+    public function up()
+    {
+        $this->forge->addField([
+            'id' => [
+                'type' => 'INT',
+                'unsigned' => true,
+                'auto_increment' => true,
+            ],
+            'time_created' => [
+                'type' => 'DATETIME',
+                'default' => new RawSql('CURRENT_TIMESTAMP'),
+                'null' => false,
+            ],
+            'time_updated' => [
+                'type' => 'DATETIME',
+                'null' => false,
+                'default' => new RawSql('CURRENT_TIMESTAMP'),
+                'on_update' => new RawSql('CURRENT_TIMESTAMP'),
+            ],
+        ]);
+        $this->forge->addKey('id', true);
```

Comment:
consider using `addPrimaryKey()` instead to make clear its a primary key.

Reply:
Implemented the change, thanks! :)